src/go/plugin/go.d/config/go.d/snmp.profiles/default/synology-disk-station.yaml

```diff
@@ -72,60 +101,56 @@ metrics:
       name: synology.system.temperature
       OID: 1.3.6.1.4.1.6574.1.2.0
       chart_meta:
-        description: The temperature of Disk Station uses Celsius degree
-        family: 'System/Temperature/Value'
+        description: The temperature of Disk Station
+        family: "System/Temperature/Value"
```

Comment:
I made it single quotes on purpose - different highlighting colour. Please undo.

<img width="1114" height="356" alt="Image" src="https://github.com/user-attachments/assets/92943722-18eb-4053-8c73-a715a5fa7d18" />

Reply:
ah, but formatting doesn't change them? I need to check my settings